data/human/ships.txt

```diff
@@ -2293,6 +2293,76 @@ ship "Leviathan"
 
 
 
+ship "Mammoth"
+	sprite "ship/mammoth"
+	thumbnail "thumbnail/mammoth"
+	attributes
+		category "Utility"
+		"cost" 18800000
+		"shields" 7800
+		"hull" 7740
+		"required crew" 18
+		"bunks" 30
+		"mass" 688
+		"drag" 10.5
+		"heat dissipation" .32
+		"fuel capacity" 1000
+		"cargo space" 550
+		"outfit space" 572
+		"weapon capacity" 288
+		"engine capacity" 129
+		"scan interference" 2
+		weapon
+			"blast radius" 140
+			"shield damage" 1400
+			"hull damage" 780
+			"hit force" 2100
+	outfits
+		"Sidewinder Launcher" 2
+		"Sidewinder Missile" 90
+		"Proton Turret" 4
+		"Heavy Anti-Missile Turret" 4
+		
+		"Fission Reactor"
+		"LP144a Battery Pack"
+		"D41-HY Shield Generator"
+		"Large Radar Jammer"
+		"Cooling Ducts"
+		"Laser Rifle" 12
+		
+		"A370 Atomic Thruster"
+		"A375 Atomic Steering"
+		"Caldera Afterburner"
+		"Scram Drive"
+		
+	engine 17 147 0.8
+	engine -17 147 0.8
+	engine -34 143 0.8
+	engine 34 143 0.8
+	engine -56 148.5 0.6
+	engine 56 148.5 0.6
+	gun -47.5 -57.5
```

Comment:
```suggestion
	gun -47.5 -57.5 "Sidewinder Missile Launcher"
```

Reply:
This is unnecessary especially if there's only single outfit to fit in the ports.

Reply:
Yeah I never do this either

Reply:
I suppose that is fair. If it was using any other gun mounts I'd say we should at that point.